osu.Game/Graphics/Backgrounds/BeatmapBackgroundWithStoryboard.cs

```diff
@@ -1,17 +1,25 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+#nullable enable
+
 using osu.Framework.Allocation;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Timing;
 using osu.Game.Beatmaps;
+using osu.Game.Overlays;
 using osu.Game.Storyboards.Drawables;
 
 namespace osu.Game.Graphics.Backgrounds
 {
     public class BeatmapBackgroundWithStoryboard : BeatmapBackground
     {
+        private InterpolatingFramedClock storyboardClock = null!;
```

Comment:
I have, actually:

https://github.com/ppy/osu/blob/f7715b930fd493323c8577f10c889bad8d1fd092/osu.Game/Beatmaps/Drawables/Cards/BeatmapCardNormal.cs#L34

Can move to ctor but it's going to be a bit finicky because the track is not available in ctor yet, so I'd have to `ChangeSource()` a second time in BDL.

Reply:
Or just make it nullable. Would want @smoogipoo's opinion on this, not sure if we should treat BDL load methods as equivalent of DI acquisition in terms of null-suppression.

Reply:
Isn't referring to `Beatmap` inside `load()` error-prone anyway? Seems like it'd be safer to `ChangeSource()` in `LoadComplete` (and also process once I guess, because I don't believe that's done by default), at which point this becomes moot.

Highly prefer not to use `null!` unless absolutely unavoidable (DI).

Reply:
I've rewritten this to avoid both the `null!` and the `Beatmap` access in `load()`. Let me know if the latest changes match your expectations.